File: fixtures/elixir_output/get_with_data2.ex
request = %HTTPoison.Request{
  method: :put,
  url: "https://api.newrelic.com/v2/alerts_policy_channels.json",
  options: [],
  headers: [
    {~s|X-Api-Key|, ~s|{admin_api_key}|},
    {~s|Content-Type|, ~s|application/json|},
  ],
  params: [
    {~s|policy_id|, ~s|policy_id|},
    {~s|channel_ids|, ~s|channel_id|},
  ],
  body: ""
}

response = HTTPoison.request(request)

File: fixtures/python/get_with_data2.py
import requests

headers = {
    'X-Api-Key': '{admin_api_key}',
    'Content-Type': 'application/json',
}

params = [
    ('policy_id', 'policy_id'),
    ('channel_ids', 'channel_id'),
]

response = requests.put('https://api.newrelic.com/v2/alerts_policy_channels.json', headers=headers, params=params)
